File: Java/Assignment 6/TestingSystem_Assignment_6/src/com/vti/entity/Ex2/Department.java
package com.vti.entity.Ex2;

public class Department {
	public static int COUNT = 0;
	private int id;
	private String name;
	public int getId() {
		return id;
	}
	public Department(String name) {
		COUNT++;
		this.id = COUNT;
		this.name = name;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public Department(int id, String name) {
		super();
		this.id = id;
		this.name = name;
	}
	@Override
	public String toString() {
		return "Department \nId: " + id + "\nName: " + name;
	}
	
	
}

File: ReactJS/TypeScript/LAB2_TypeScript/Department.js
"use strict";
exports.__esModule = true;
exports.printDepartment = exports.Department = void 0;
var Department = /** @class */ (function () {
    function Department(id, name) {
        this.id = id;
        this.name = name;
    }
    Department.prototype.getName = function () {
        return this.name;
    };
    Department.prototype.setName = function (v) {
        this.name = v;
    };
    Department.prototype.getId = function () {
        return this.id;
    };
    Department.prototype.setId = function (v) {
        this.id = v;
    };
    return Department;
}());
exports.Department = Department;
function printDepartment(department) {
    console.log("ID: ", department.getId(), "Name: ", department.getName());
}
exports.printDepartment = printDepartment;
